pkg/apiserver/apiserver.go

```diff
@@ -213,7 +214,7 @@ func (s *APIServer) installMetricsAPI() {
 
 // Install all kubesphere api groups
 // Installation happens before all informers start to cache objects, so
-//   any attempt to list objects using listers will get empty results.
+//	any attempt to list objects using listers will get empty results.
```

Comment:
```suggestion
//   any attempt to list objects using listers will get empty results.
```
Same here.

Reply:
Sorry, there should be a problem with some indentation of the computer.